drivers/SPI.h

```diff
@@ -77,22 +82,22 @@ class SPI : private NonCopyable<SPI> {
 
 public:
 
-    /** Create a SPI master connected to the specified pins
+    /** Create a SPI master connected to the specified pins.
      *
-     *  mosi or miso can be specified as NC if not used
+     *  @note Either mosi or miso can be specified as NC if not used.
```

Comment:
Not sure what device-side standards are, but we usually replace passive constructions like this with 'you can'.